src/core/store/modules/theme/theme.reducer.ts

```diff
@@ -15,6 +15,8 @@ const themeReducer = createReducer(themeInitialState, (builder) => {
   builder.addCase(changeTheme.fulfilled, (state, { payload: { theme } }) => {
     // TODO Check that the theme exists on AVAILABLE_THEMES or AVAILABLE_CUSTOM_THEMES
     state.current = theme;
+    //Adding the current theme to the dataset of body, in order to use it with css queries
+    document.body.dataset.theme = theme;
```

Comment:
Put the currently selected theme in `data-theme` of the body. Used to track the current theme.

Reply:
should we better set it dynamically in the global styles using the theme selector ? 

Is not a good practice to modify other values out of the scope of the reducer (see docs [here](https://redux.js.org/style-guide/#reducers-must-not-have-side-effects))

Reply:
I found it like the easiest way to be sure the data is always updated on theme change. I would be happy to change that, but do you have an _exact_ idea where we should put it ? 😅

Reply:
Yeah, we can place it in the action for now, as its a better place for it. 

Maybe the best place would be moving these [GlobalStyle](https://github.com/yearn/yearn-finance-v3/blob/develop/src/client/index.tsx#L82) component into the [Themable](https://github.com/yearn/yearn-finance-v3/blob/develop/src/client/containers/Themable/index.tsx) container, so that we can access the current selected theme state, and pass it down as a prop to be used in the html body. We can spend more time later on that refactor though.